content/03.methods.md

```diff
@@ -106,10 +106,23 @@ Per recommendations by the New York Genome Center [@doi:10.1101/623702], the Lan
 
 We filtered outputs from both callers on the "PASS" filter, and annotated using The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, and created MAFs using MSKCC's vcf2maf [@url:https://github.com/mskcc/vcf2maf] v1.6.17.
 
+#### Consensus SNV Calling
+
+We considered mutations to be the same across callers if they were identical for the following MAF fields: `Chromosome`, `Start_Position`, `Reference_Allele`,  `Allele`, and `Tumor_Sample_Barcode`.
+We compared all four callers in their overlap with each other, variant allele frequencies of their calls, base changes called, and genomic regions of their calls [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/compare_snv_callers_plots.Rmd].
+VarDict called numerous variants with low allele frequency [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/vaf_violin_plot.png] in addition to nearly all of the calls identified by the other three callers [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/upset_plot.png].
+Because our filtered set was based on the intersection, we removed it from the consensus calculation.
+Strelka2 does not call multinucleotide variants (MNV), but instead calls each component SNV as a separate mutation, so we separated MNV calls from Mutect2 and Lancet into consecutive SNVs before comparing them with Strelka2.
+We termed mutations shared among Strelka2, Mutect2, and Lancet consensus mutation calls.
+
+For downstream analyses, only exonic SNVs (based on Gencode v27 [@url:https://www.gencodegenes.org/human/release_27.html]) are used, to enhance comparability to other studies.
```

Comment:
Is this for all downstream analyses or only the TMB ones? Just trying to decide if it goes here or in the TMB section.

Reply:
I was thinking also for mutational signatures we should stick to exonic so that's why I wrote it this way, but currently that is not just exonic. I'd have to think about this some more though.